lib/ModelMixins/GeojsonMixin.ts

```diff
@@ -614,48 +617,47 @@ function GeoJsonMixin<T extends Constructor<Model<GeoJsonTraits>>>(Base: T) {
       return CzmlDataSource.load(rootCzml);
     }
 
-    /** Note, this is not reactive
-     * To re-style geoJSON you will need to call `loadMapItems` again
-     */
-    @action
-    private stylesWithDefaults() {
-      const style = this.style;
-
+    @computed
+    get stylesWithDefaults() {
       const options = {
         describe: describeWithoutUnderscores,
-        markerSize: parseMarkerSize(style["marker-size"]) ?? 20,
-        markerSymbol: style["marker-symbol"], // and undefined if none
-        markerColor: defaultColor(style["marker-color"], this.name ?? ""),
-        stroke: getColor(style.stroke ?? "#000000"),
-        polygonStroke: getColor(style.stroke ?? "#000000"),
-        polylineStroke: defaultColor(style.stroke, this.name || ""),
-        strokeWidth: style["stroke-width"] ?? 2,
+        markerSize: parseMarkerSize(this.style["marker-size"]) ?? 20,
```

Comment:
an optional suggestion would be to have a `const DEFAULTS = {...}` object with the same type as this options object. That way, we can ensure that all the fields in option have a fallback value and it separates out the default value from the fallback logic a bit.

Reply:
Yeah good idea!